packages/webapp/src/config.ts

```diff
@@ -115,6 +115,10 @@ export const validUploadActions: ValidUploadActions = {
     },
 };
 
+export const electionMeetingDurationMs =
+    Number(process.env.NEXT_PUBLIC_ELECTION_MEETING_DURATION_MS) ||
```

Comment:
please add `NEXT_PUBLIC_ELECTION_MEETING_DURATION_MS` in the above log and variable check, which will make you to add to .env file too. (It will help us on deploying/debugging in another environments)

Reply:
In the interest of time, I'll go ahead and do this. But are you opposed to the idea of something like this in the logs?

`ELECTION_MEETING_DURATION_MS="${process.env.NEXT_PUBLIC_AA_FETCH_AFTER || "[NOT SET - DEFAULTING TO 40 MIN]"}"`

That way, we don't have to keep adding to the already sizable number of env vars?

Reply:
(And good catch, btw. Def needs to be logged out.)